test/requirements.txt

```diff
@@ -4,6 +4,7 @@ boto3==1.28.53
 certifi>=2023.7.22
 invoke==2.2.0
 mock
+pennylane
```

Comment:
Added because it is used in the entry point script. Not necessarily a new constraint from decorator jobs-- we'd need to do this if the entry point was a function too (vs a file)

Reply:
Where is pennylane used in the entry point script?

Reply:
Pennylane is globally imported in the file defining the customer entry point function, meaning it needs to be installed when we import the function. We can prevent this by importing PL within the job scope (inside the entry point function), but that would be messier and involve refactoring some of our helpers. In practice it's incredibly unlikely imo that customers are writing PL jobs without having PL locally.

The reason we didn't need this previously is because we were using a whole file as an entry point script (`__main__` method). Even with the old interface, if we had instead chosen a specific function to use for entry point, we would have run into this same issue.